beetle_tester.py

```diff
@@ -0,0 +1,45 @@
+import arcade
+from beetles import Beetle
+from teams import Team
+from team_color import TeamColor
+
+# Constants
+SCREEN_WIDTH = 1280
+SCREEN_HEIGHT = 720
+SCREEN_TITLE = "Welcome to the Ouch, Motherfucker"
```

Comment:
We should probably give the tester a different title.

Reply:
done